CHANGELOG.md

```diff
@@ -40,6 +40,11 @@
   - `POST <legacy-http-prefix>/push`
   - `POST /push`
   - `POST /ingester/push`
+* [CHANGE] Renamed CLI flags to configure the network interface names from which automatically detect the instance IP. Not a breaking change because these flags were non documented. #3295
```

Comment:
```suggestion
* [CHANGE] Renamed CLI flags to configure the network interface names from which automatically detect the instance IP. Not a breaking change because these flags were not documented. #3295
```

(or `undocumented`)

Reply:
There were documented in -help, I was using them in tests.

Reply:
You're right. `-help` shows everything. Removing that sentence.